ballerina/build.gradle

```diff
@@ -859,7 +946,8 @@ unpackDockerAnnotations.dependsOn unpackAwsLambdaExamples
 unpackDockerExamples.dependsOn unpackDockerAnnotations
 unpackKubernetesAnnotations.dependsOn unpackDockerExamples
 unpackKubernetesExamples.dependsOn unpackKubernetesAnnotations
-unpackBalCommand.dependsOn unpackKubernetesExamples
+unpackStdLibs.dependsOn unpackKubernetesExamples
+unpackBalCommand.dependsOn unpackStdLibs
```

Comment:
@BuddhiWathsala Just wondering how we are going to pack api docs into distribution? 
Also socket BBE are we keeping it on balllerina-lang repository?

Reply:
The current idea is to pack the API docs along with the module zip file and use it here. I'm currently working on it.

Reply:
How about considering API docs as a separate task https://github.com/ballerina-platform/ballerina-lang/issues/23506